topics/admin/tutorials/tool-management/tutorial.md

```diff
@@ -36,15 +36,26 @@ This tutorial will introduce you to one of Galaxy's associated projects - [Ephem
 
 # Background
 
-You are an administrator of your lab's Galaxy. A colleague has approached you with a request to run a specific [Galaxy workflow]({% link topics/sequence-analysis/tutorials/mapping/workflows/mapping.ga %}) on the lab's data. In order to run this workflow you have to accomplish several substeps first. You will need to:
+You are an administrator of a Galaxy server. A colleague has approached you with a request to run a specific [Galaxy workflow]({% link topics/sequence-analysis/tutorials/mapping/workflows/mapping.ga %}) on their data. In order to enable this workflow for your users, you will have to:
 
 - identify what tools are required for the workflow
-- and install these on your Galaxy instance
+- install these tools and their dependencies on your Galaxy instance.
 
 
 # Requirements
 
-To run this tutorial, first you will need to [install Ephemeris](https://ephemeris.readthedocs.io/en/latest/installation.html). We recommend installing it in a virtualenv.
+To run this tutorial, you will need to [install Ephemeris](https://ephemeris.readthedocs.io/en/latest/installation.html). You would normally install it on your workstation, but during training courses we recommend to install it on the same virtual machine used for the Galaxy server.
+
+> ### {% icon tip %} Installing Ephemeris in a Python virtual environment
+>
+> - Install `virtualenv` if it is not already available. On Ubuntu this can be done with `sudo apt install virtualenv`
+> - Create a virtual environment just for ephemeris, activate it and install ephemeris inside it:
+>   ```console
+>   virtualenv -p python3 ephemeris_venv
```

Comment:
any chance this could be: 

```suggestion
>   python3 -m virtualenv -p python3 ephemeris_venv
```

Reply:
What's the advantage? It's longer and not how it's explained in most docs I've encountered.